src/MessageProperty/Conventions.cs

```diff
@@ -25,6 +25,11 @@ public bool IsEncryptedProperty(PropertyInfo property)
             }
         }
 
-        internal Func<PropertyInfo, bool> IsEncryptedPropertyAction = p => typeof(WireEncryptedString).IsAssignableFrom(p.PropertyType);
+        /// <summary>
+        /// The convention.
+        /// </summary>
```

Comment:
What happens when `WireEncryptedString` becomes obsolete? Do we need to disable the warning somewhere?